src/puptoo/process/profile.py

```diff
@@ -644,6 +650,42 @@ def system_profile(
     if gcp_network_interfaces:
         profile["public_ipv4_addresses"] = _remove_empty_string(gcp_network_interfaces.public_ips)
 
+    if iris_cpfs and iris_list:
+        try:
+            iris_profile = {}
```

Comment:
Before merging this PR, we would also be required to add a schema 
under inventory-schema repo [like this](https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L413). Please have a look.

Reply:
Thanks for the comments @r14chandra . Adding the related schema to inventory-schema repo at https://github.com/RedHatInsights/inventory-schemas/pull/122 .